react/Viewer/helpers.js

```diff
@@ -168,3 +169,31 @@ export const isEditableAttribute = (name, file) => {
     ((name === 'issueDate' && !isFromKonnector(file)) || name !== 'issueDate')
   )
 }
+
+export const normalizeAndSpreadAttributes = rawFile => {
+  const normalizedFile = normalize(rawFile)
+
+  return {
+    ...normalizedFile,
+    ...normalizedFile?.attributes
+  }
+}
+
+export const removeFilename = path => /(.+)(\/.+)$/.exec(path)?.[1] || path
```

Comment:
You have the real filename where you call this method. You should pass it to this method along side with the path. And then you just have to remove the all string instead of doing regex?

Reply:
Yes, this was a copy and paste from cozy-notes without thinking about it. Now, I use `.replace` instead of an regex